docs/source/api/deliveryservices_id_routing.rst

```diff
@@ -21,12 +21,17 @@
 
 ``GET``
 =======
-Retrieves routing method statistics for a particular :term:`Delivery Service`
+Retrieves the aggregated routing percentages for a given :term:`Delivery Service`. This is accomplished by polling stats from all online Traffic Routers via the ``/crs/stats`` route.
 
 :Auth. Required: Yes
 :Roles Required: None\ [#tenancy]_
 :Response Type:  Object
 
+.. versionchanged:: 1.5
+	Older versions of this endpoint had different behaviors in handling non HTTP/DNS :term:`Delivery Services` and :term:`Delivery Service` regexps.
+	It used to return an empty response object if the :term:`Delivery Service` being requested did not have a HTTP or DNS type. It will now return an error.
+	For :term:`Delivery Service` regexps it used to use all regexps to compare host names in the routing stats which was incorrect. It will now only compare :term:`Delivery Service` host regexps.
```

Comment:
Can you please spell out "regular expression(s)"?

Reply:
🖖